### PR TITLE
Add accessibility test to signin widget js selenium test.

### PR DESCRIPTION
**Pull request by jhe-okta**
axe-selenium-js-2

### CODE REVIEW

File: test/e2e/specs/OIDC_spec.js [hunk 1]
```diff
@@ -53,6 +54,7 @@ describe('OIDC flows', function() {
           }
         ]
       });
+      Expect.toBeA11yCompliant();
```
Comment (rchild-okta) [inline test/e2e/specs/OIDC_spec.js]: Where is this function defined?

**Comment by alanzhou-okta** [comment]
Also, you're missing a ':' at the end of your emoji

File: test/e2e/util/Expect.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+var Expect = module.exports = {};
+
+var _ = require('lodash'),
+    grunt = require('grunt'),
+    axe = require('../../../node_modules/axe-webdriverjs'),
```
Comment (rchild-okta) [inline test/e2e/util/Expect.js]: interesting - can't we just do
```javascript
axe = require('axe-webdriverjs');
```

File: test/e2e/util/Expect.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+              help      = violation.help;
+          grunt.log.writeln();
+          grunt.log.writeln('    ' + severity + ': ' + help);
+          _.each(violation.nodes, function (node) {
+            var htmlsnippet = node.html;
+            grunt.log.writeln('    - ' + htmlsnippet);
+          });
+        });
+      } else {
+        grunt.log.subhead('  There is no accessibility violation');
```
Comment (rchild-okta) [inline test/e2e/util/Expect.js]: nit: "are no accessibility violations"

**Pull request merged**